core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunctionTest.java

```diff
@@ -1570,6 +1570,50 @@ public void pow_null_missing() {
     assertTrue(power.valueOf(valueEnv()).isMissing());
   }
 
+  /**
+   * Test pow/power with null output.
+   */
+  @Test
+  public void pow_null_output() {
+    FunctionExpression pow = DSL.pow(DSL.literal((double) -2), DSL.literal(1.5));
+    assertEquals(pow.type(), DOUBLE);
+    assertEquals(String.format("pow(%s, %s)", (double) -2, 1.5), pow.toString());
+    assertTrue(pow.valueOf(valueEnv()).isNull());
+
+    pow = DSL.pow(DSL.literal((double) -2), DSL.literal((double) 2));
+    assertEquals(pow.type(), DOUBLE);
+    assertEquals(String.format("pow(%s, %s)",(double) -2, (double) 2), pow.toString());
+    assertThat(
+            pow.valueOf(valueEnv()),
+            allOf(hasType(DOUBLE), hasValue(Math.pow(-2, 2))));
```

Comment:
I think cases that have non-null results should be in a different test or removed if they already exist

Reply:
Separated tests in `16661a0`